src/Kafka/Settings/Settings.tsx

```diff
@@ -0,0 +1,157 @@
+import {
+  Button,
+  Card,
+  CardBody,
+  Flex,
+  FlexItem,
+  Modal,
+  ModalVariant,
+  PageSection,
+  Spinner,
+  Switch,
+} from "@patternfly/react-core";
+import type { FunctionComponent } from "react";
+import { useTranslation } from "react-i18next";
+import { SettingsAlert } from "./components";
+import type { AlertStatus, SettingsStatus } from "./types";
+import "./Settings.css";
+
+export type SettingsProps = {
+  connectionStatus: SettingsStatus;
+  onSwitchClick: () => void;
+  isModalOpen: boolean;
+  onClickTurnOff: () => void;
+  alertStatus: AlertStatus;
+  connectionState: boolean;
+  onClickClose: () => void;
+};
+
+export const Settings: FunctionComponent<SettingsProps> = ({
+  connectionStatus,
+  onSwitchClick,
+  isModalOpen,
+  onClickTurnOff,
+  alertStatus,
+  onClickClose,
+  connectionState,
+}) => {
+  const { t } = useTranslation("kafka");
+
+  return (
+    <>
+      <PageSection>
+        <Card>
+          <CardBody>{t("settings.kafka_instance_settings_label")}</CardBody>
+        </Card>
+      </PageSection>
+      <PageSection>
+        <Card className={"settings-card--max-width"}>
+          <CardBody>
+            <Flex flexWrap={{ default: "nowrap" }}>
+              <FlexItem grow={{ default: "grow" }}>
+                <Flex>
+                  <FlexItem component={"span"} spacer={{ default: "spacerSm" }}>
+                    <strong>
+                      {t("settings.connection_re_authentication_label")}
+                      {":"}
+                    </strong>
+                  </FlexItem>
+                  <FlexItem>
+                    {(() => {
+                      switch (connectionStatus) {
+                        case "On":
```

Comment:
It would be better to create enum type for connection  statuses i.e. On. Off, Turning etc.

Reply:
I have created the type to manage connection status https://github.com/redhat-developer/app-services-ui-components/blob/a717e300f8cb2a2c3c7df45d1cdc5d4f7b5162d7/src/Kafka/Settings/types.ts#L1. Do you think we should create enum type?

Reply:
we can use this `SettingsStatus` type which is already there. No need to create enum type.

Reply:
yes, I am using that only https://github.com/redhat-developer/app-services-ui-components/blob/a717e300f8cb2a2c3c7df45d1cdc5d4f7b5162d7/src/Kafka/Settings/Settings.tsx#L20